package.json

```diff
@@ -105,6 +113,7 @@
     "nock": "^13.0.7",
     "node-sass": "^5.0.0",
     "pa11y": "^5.3.0",
+    "playwright": "^1.8.0",
```

Comment:
still needed?

Reply:
👍 Good spot, Playwright is used to run the test in Chrome, Safari and Firefox, but looks like I don't need this here since it's already a dependency of `codeceptjs`.

Reply:
👍 Fixed in 2ad33d2